pkgs/development/python-modules/python-efl/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchurl, buildPythonPackage, pkgconfig, python, enlightenment }:
+
+# Should be bumped along with EFL!
+
+buildPythonPackage rec {
+  name = "python-efl-${version}";
+  version = "1.21.0";
+
+  src = fetchurl {
+    url = "http://download.enlightenment.org/rel/bindings/python/${name}.tar.xz";
+    sha256 = "08x2cv8hnf004c3711250wrax21ffj5y8951pvk77h98als4pq47";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+
+  buildInputs = [ enlightenment.efl ];
+
+  propagatedBuildInputs = [ python.pkgs.dbus-python ];
+
+  preConfigure = ''
+    export NIX_CFLAGS_COMPILE="$(pkg-config --cflags efl) -I${python.pkgs.dbus-python}/include/dbus-1.0 $NIX_CFLAGS_COMPILE"
+  '';
+  
+  preBuild = "${python}/bin/${python.executable} setup.py build_ext";
+
+  installPhase= "${python}/bin/${python.executable} setup.py install --prefix=$out";
```

Comment:
Is this phase not the default?

Reply:
> Is this phase not the default?

I do not think so, because without it most of the files are not installed. It was already present in prior versions of the nix expression.